Added DigitalOcean & AliCloud to the README

pingrishabh opened the pull request:
Edited the README to include DigitalOcean & AliCloudDigital to the features table and appended their usage wherever necessary.

nicolasraube commented:
The only possibility I see for arguments is `npm run lint -- lib test` but then exit 0 doesn't work anymore and npm says linting failed because the exit status is 1. Any suggestions?

agentmilindu commented on package.json:
```diff
@@ -5,7 +5,8 @@
     "test": "mocha test/compute/ec2.js",
-    "coverage": "istanbul --include-all-sources cover _mocha -- -R dot --recursive test/"
+    "coverage": "istanbul --include-all-sources cover _mocha -- -R dot --recursive test/",
+    "lint": "eslint lib; eslint test; exit 0"
```
Let's remove `exit 0`. This will result in a successful exit even if there are lint errors. 

BTW, let's not hardcode `lib` and `test`. Can't we have it like `eslint $` and then run whatever we like as `npm run lint examples`, `npm run lint lib`, etc?

nicolasraube commented:
@agentmilindu Changed it. I still get this error after linting.
![captura de pantalla de 2017-12-24 10-42-53](https://user-images.githubusercontent.com/19194376/34325629-49e7f5b0-e897-11e7-8dfb-ffc62ed588fe.png)
I found this issue https://github.com/eslint/eslint/issues/2409 but they can only suggest exit 0. Ugly but I guess we have to live with it.

agentmilindu commented:
@nicolasraube The linting fails because there are linting errors, and this is the expected behaviour. 
Anyway, good job!

pingrishabh committed:
Added DigitalOcean & AliCloud to the README

Edited the README to include DigitalOcean & AliCloudDigital to the features table and appended their usage wherever necessary

rajikaimal commented on README.md:
```diff
@@ -21,29 +21,31 @@ yarn add nodecloud
+- AliCloud
+- DigitalOcean
+- Amazon Web Services (AWS)
+- Google Cloud Platform (GCP)
```
Let's order this alphabetically

pingrishabh replied:
Understood. I'll do that.

pingrishabh replied:
@rajikaimal Done!

pingrishabh committed:
Alphabetically arranged 'Service Providers'
